### DISCUSSION TITLE
Installation/Upgrade Tutorials

### DISCUSSION THREAD

**Discussion by NMinker** [mattwebbio/orbital-sync#383] [Ideas]
With Gravity-Sync being retired, I think it would be wise to add a Wiki tab to this project. Put together a tutorial on how to install Orbital Sync with Docker. There will be a bunch of people (myself included) transitioning to Orbital Sync as soon as Pi-Hole 6 is released and Orbital sync supports it.

I don't use Docker (although I messed around with it a couple years ago), have Pi-Hole installed via the automated install. I can follow the Basic installation instructions from the Docker Docs (i.e. for [Debian/arm64 install](https://docs.docker.com/engine/install/debian/)) and the docker pull command for Orbital ([from here](https://hub.docker.com/r/mattwebbio/orbital-sync)), but after that I don't know what I'm doing. I think there might be others who encounter this.

Don't want to do too much searching. Also, a FAQ for the gravity-sync to orbital-sync transitions would be useful

**Comment by jthombenj** [comment]
Did you ever figure this out? I've been playing with it for a couple days and can't get anywhere. I edited the docker-compose.yaml with the configurations as specified but I still get an error that they aren't set. I'm sure I'm doing something wrong, but info isn't easy to find and most comments aren't helpful.
